hack4impact-utk/compassion-ministries
# upsert volunteer on checkin

# Description

We can now intentionally edit volunteers during the check in flow :)

## Relevant issue(s)



closes #213 

## Questions


## Type of change




# Testing steps
try and break it, check in all kinds of vols, those that need role verification, new vols, edited vols, normal non-edited vols, and combinations where you can

# Checklist:
